Exercise_E.js

```diff
@@ -34,7 +34,18 @@ var KEYS = {
 };
 

```

Comment:
Great pseudo code here and fantastic implementation of the maya function, you are the only one who has used concat here! Well done!